pages/linux/isort.md

```diff
@@ -0,0 +1,37 @@
+# isort
+
+> Sort Python import statements alphabetically within sections and by type.
+> `isort` can be run on multiple files or an entire project, simplifying import management.
+> More information: <https://pycqa.github.io/isort/>.
+
+- Sort imports in a Python file, modifying the file in-place:
+
+`isort {{path/to/file.py}}`
+
+- Sort imports in multiple Python files, modifying the files in-place:
+
+`isort {{path/to/file1.py}} {{path/to/file2.py}} ...`
```

Comment:
```suggestion
`isort {{path/to/file1.py path/to/file2.py ...}}`
```

Please see this section in https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#grouping-placeholders